GUIDE.md

```diff
@@ -255,3 +258,13 @@ By default, when executed, the run.sh script will generate a data file named `sh
 If necessary, you can choose an alternative location for data storage by specifying the full path of the file as the value of the `--storage.files` flag in the run.sh script.
 
 Please refer to [configuration](/README.md#configuration) for more details.
+
+### What can I do about "The zkey file was not downloaded" error?
+
+When you see the following message when running **run.sh**. you can manually download the blob_poseidon.zkey/blob_poseidon2.zkey to `./build/bin/snarkjs/` folder and run it again. 
```

Comment:
You should probably tell the user the link to download manually:
https://drive.google.com/file/d/1olfJvXPJ25Rbcjj9udFlIVr08cUCgE4l/view